src/system.c

```diff
@@ -653,15 +653,15 @@ static const struct optInfo options[] = {
   { 'E', "", toggle, &SyUseReadline, 0 }, /* kernel */
   { 'l', "roots", setGapRootPath, 0, 1}, /* kernel */
 #ifdef USE_GASMAN
-  { 'm', "", storeMemory2, &SyStorMin, 1 }, /* kernel */
+  { 'm', "initial-workspace", storeMemory2, &SyStorMin, 1 }, /* kernel */
 #endif
   { 'r', "", toggle, &IgnoreGapRC, 0 }, /* kernel */
 #ifdef USE_GASMAN
   { 's', "", storeMemory, &SyAllocPool, 1 }, /* kernel */
 #endif
   { 'n', "", forceLineEditing, 0, 0}, /* prob library */
 #ifdef USE_GASMAN
-  { 'o', "", storeMemory2, &SyStorMax, 1 }, /* library with new interface */
+  { 'o', "warn-workspace", storeMemory2, &SyStorMax, 1 }, /* library with new interface */
```

Comment:
off topic: looking at this, I really wonder what the sort order of this struct is? In parts it looks as if it is sorted alphabetically by the short name, upper case before lower case; but that's not really true, as e.g. `E` comes between `f` and `l`... Huh.

Seems to be me we should either sort like that, or else "group logically" / "group as in system,g" ?

Again: this is not a change request for this PR, I am just wondering out loud